src/libraries/Common/tests/TestUtilities/System/IO/FileCleanupTestBase.cs

```diff
@@ -214,5 +214,25 @@ private unsafe uint GetFinalPathNameByHandle(SafeFileHandle handle, char[] buffe
                 return Interop.Kernel32.GetFinalPathNameByHandle(handle, bufPtr, (uint)buffer.Length, Interop.Kernel32.FILE_NAME_NORMALIZED);
             }
         }
+
+        protected string CreateTestDirectory(params string[] paths)
+        {
+            string dir = Path.Combine(paths);
+            Directory.CreateDirectory(dir);
```

Comment:
```suggestion
            Assert.True(Path.IsPathRooted(file));
            Directory.CreateDirectory(dir);
```
this ensures we're not depending on whatever the current directory is -- the path is rooted.